src/core/core.controller.js

```diff
@@ -240,6 +249,11 @@ module.exports = function(Chart) {
 					return;
 				}
 
+				if (positionIsHorizontal(scaleOptions.position) !==
```

Comment:
[minor] can we keep the condition on the same line, quite confusing when reading :) ?